### DISCUSSION TITLE
Why no release since Jan 10?

### DISCUSSION THREAD

**Discussion by rupeshknn** [google/jaxopt#595] [General]
Hey guys, I'm new to JaxOpt but familiar with Jax. There are some merged PRs in `main` that I'd like in my environment, but it seems the last release was on Jan 10. It's been 4 months now. Is there a reason why there have been no releases?

**Comment by fabianp** [comment]
we're not actively working on jaxopt anymore. We're focusing on migrating some of its features to optax

**Reply by rupeshknn** [reply]
I get that, but then the PRs that have already been merged could be released right?

I don't know a lot about the release processes of such projects but I'm assuming it's just a few minutes of work? I apologize if that's incorrect.

**Reply by mblondel** [reply]
I'll do a release next week.

**Reply by rupeshknn** [reply]
hi @mblondel just following up on this
